semver.md

```diff
@@ -61,8 +61,8 @@ could be declared in the code itself or exist strictly in documentation.
 However it is done, it should be precise and comprehensive.
 
 1. A normal version number MUST take the form X.Y.Z where X, Y, and Z are
-non-negative integers, and MUST NOT contain leading zeroes. X is the
-major version, Y is the minor version, and Z is the patch version.
+non-negative integers. Elements (X, Y and Z) MUST NOT contain leading zeroes.
+X is the major version, Y is the minor version, and Z is the patch version.
```

Comment:
How about just changing it from the "...MUST NOT contain leading zeroes..." in the original as:

...except for a single character ('0') place holder, MUST NOT contain leading zeroes.

Reply:
This is implied. A single `0` number does not have leading zeroes since the single zero digit is required here. This is universally understood in all other occasions I've seen this being discussed in one way or another.